src/AutoInstrumentationInstaller/install-otel-instrumentation

```diff
@@ -0,0 +1,462 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+// This simple CLI script installs and setup
+// everything what's needed for auto-instrumentation.
+
+// default http async implementation
+// in basic mode
+$dependencies = array(
+  "php-http/guzzle7-adapter"
+);
+
+// core packages are always installed
+// it's possible to get them via packagist API
+// however there is no way at the moment
+// to distinguish core packages from others
+$opentelemetry_core_packages = array(
+  "open-telemetry/sdk", 
+  "open-telemetry/api",
+  "open-telemetry/sdk-contrib",
```

Comment:
this is a package we've just deprecated. How about just choosing `open-telemetry/exporer-otlp`? (Optionally, with the grpc transport and zipkin exporter, too?)

Reply:
Yes, that good point. So, for now I will add `open-telemetry/exporer-zipkin` by default.